README.md

```
@@ -1,134 +1,259 @@
+- Visualization: Recharts
+- State Management: Redux Toolkit, TanStack Query
+
+## 📂디렉토리 구조
+- apps
+    - web: 사용자 서비스
+    - admin: 관리자 서비스
+- packages
+    - 두 서비스의 공통 컴포넌트, 함수 등
+
+### apps(실 서비스 단) 내부 구조
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)